t/t205-event-locking.t

```diff
@@ -25,9 +25,6 @@ given path.
 
 projfs_start test_handlers source target --timeout 1 --lock-file lock || exit 1
 
-# wait_mount will trigger a projection, so we need to reset it to empty
-setfattr -n user.projection.empty -v y source
-
```

Comment:
Lovely to see this go!

Reply:
![Earl grey?](https://user-images.githubusercontent.com/1915/58296558-84de0080-7e17-11e9-8c0c-29f3ef66903d.jpg)